clap_complete/src/shells/bash.rs

```diff
@@ -58,7 +58,7 @@ impl Generator for Bash {
     esac
 }}
 
-complete -F _{name} -o bashdefault -o default {name}
+complete -F _{name} -o nosort -o bashdefault -o default {name}
```

Comment:
Yea, that's why I'm looking into nix. If done correctly and sort order is deterministic - it will ensure the sorting order to be the same locally and remote.